starter-repos/app-template/skaffold-ci.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: skaffold/v2beta7
+kind: Config
+metadata:
+  name: app-template
+build:
+  artifacts:
+  # Make sure the skaffold "default-repo" is set.
+  # Follow the doc for instructions https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/master/docs/index.md#testing-the-change-locally-with-skaffold 
+  - image: app
+profiles:
+  - name: staging-central
+    deploy:
+      kubectl:
+        manifests:
+        - config/envs/staging-central/deployment.yaml
+        flags:
+          disableValidation: true
```

Comment:
Why do we disable validation for kubectl deployer?

Reply:
Because the <ENV>/deployment.yaml is actually the kustomization overlay. it fails the validation check. skaffold render is only needed to add the real image digest, it doesn't need to have the full config.